stable/openldap/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- enis
```

Comment:
while this is certainly not a blocker, I am open to suggestions on _if_ this PR should add approvers/maintainers and who should act as such.